airflow/www/static/js/base.js

```diff
@@ -47,57 +48,56 @@ function changDisplayedTimezone(tz) {
   });
 }
 
-var el = document.createElement('span');
-
 export function escapeHtml(text) {
+  const el = document.createElement('span');
   el.textContent = text;
   return el.innerHTML;
 }
 
 window.escapeHtml = escapeHtml;
 
-export function convertSecsToHumanReadable(seconds) {
-  var oriSeconds = seconds;
-  var floatingPart = oriSeconds- Math.floor(oriSeconds);
+export function convertSecsToHumanReadable(secs) {
+  const oriSeconds = secs;
```

Comment:
Do we have Babel on? I know there were some compatibility issues with older browsers.